src/open_inwoner/openklant/services.py

```diff
@@ -1041,6 +1041,20 @@ def find_organisatie_for_kvk_and_vestiging(
 
         return self.client.partij.retrieve(vestiging_pi["identificeerdePartij"]["uuid"])
 
+    def get_partij_for_user(self, user: User) -> Partij | None:
+        if user.kvk:
+            if organisatie := self.find_organisatie_for_kvk_and_vestiging(
+                kvk=user.kvk,
+                vestigingsnummer=user.vestiging,
+            ):
+                return organisatie
+
+        if user.bsn:
+            if persoon := self.find_persoon_for_bsn(user.bsn):
```

Comment:
```suggestion
            return self.find_persoon_for_bsn(user.bsn)
```
See previous comment